src/lib/insights/client.ts

```diff
@@ -34,7 +34,9 @@ const getQueryID = (selectedHits: Hits): string => {
   const queryID = queryIDs[0];
   if (typeof queryID !== 'string') {
     throw new Error(
-      'Could not infer `queryID`. Ensure InstantSearch is configured with `clickAnalytics: true`'
+      `Could not infer \`queryID\`. Ensure InstantSearch \`clickAnalytics: true\` was added with the Configure widget.
+
+See: https://alg.li/lNiZZ7`
```

Comment:
I normally make a  rememberable link, but I think we can do a redo on the url minification another time (something with identifiers and stored together with the code so it can be tested?)

Reply:
I thought the same but I was undecided on the "rememberable" part as it has to be explicit and short. Maybe you can help:
How about `https://alg.li/click-analytics` ?

Reply:
Makes sense, I don't think `click-analytics` fits that well here, so let's think about generalising these urls another time

Reply:
ok, works for me!